openssl/src/crypto/pkey.rs

```diff
@@ -93,6 +98,33 @@ impl PKey {
         }
     }
 
+    /// Read a private key from PEM, supplying a password callback to be invoked if the private key
+    /// is encrypted.
+    ///
+    /// The callback will be passed the password buffer and should return the number of characters
+    /// placed into the buffer.
```

Comment:
The docs for these methods should mention that they require the `catch_unwind` feature.